templates/taxi/manufacturer_list.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
+    <h1>Manufacturer List <a href="{% url 'taxi:manufacturer-create' %}">+</a>
```

Comment:
Code Style Issue: The link to create a new manufacturer is not clearly labeled. Consider adding a clear label for the link, such as 'Add New Manufacturer', instead of just '+'. This improves the user interface and makes the website more user-friendly.